crates/integration/src/lib.rs

```diff
@@ -445,6 +534,142 @@ edition.workspace = true
             .to_string()
     }
 
+    pub fn set_version(&self, version: &str, push: bool) {
+        let mut command = std::process::Command::new("cargo");
+
+        command
+            .current_dir(self.temp_dir.path())
+            .arg("workspaces")
+            .arg("version");
+
+        if !push {
+            command.arg("--no-git-push");
+        }
+
+        command
+            .arg("--no-individual-tags")
+            .arg("--yes")
+            .arg("custom")
+            .arg(version.strip_prefix("v").unwrap_or(version))
```

Comment:
This looks like [`trim_start_matches`](https://doc.rust-lang.org/std/primitive.str.html#method.trim_start_matches), it could also use a `char` instead of a `str`, which I assume is a simpler implementation under the hood.

```suggestion
            .arg(version.trim_start_matches('v'))
```